test/integration/targets/vmware_guest/defaults/main.yml

```diff
@@ -29,4 +29,4 @@ vmware_guest_test_playbooks:
   - poweroff_d1_c1_f1.yml
 #  - template_d1_c1_f0.yml
   - vapp_d1_c1_f0.yml
-  - reconfig_vm_to_latest_version.yml
```

Comment:
No need to remove existing test case.

Reply:
Oops! Sorry by mistake I removed the testcase. Thought it is in progress PR Testcase
Added it back